test.ts

```diff
@@ -58,4 +58,51 @@ myKVNamespace.get("foo", {type: "json"});
 myKVNamespace.get("foo", {cacheTtl: 60, type: "arrayBuffer"});
 myKVNamespace.get("foo", {type: "arrayBuffer"});
 myKVNamespace.get("foo", {cacheTtl: 60, type: "stream"});
-myKVNamespace.get("foo", {type: "stream"});
\ No newline at end of file
+myKVNamespace.get("foo", {type: "stream"});
+
+interface Env extends ModuleWorker.Bindings {
+  Clients: KVNamespace;
+  Workers: DurableObjectNamespace;
+  API_TOKEN: string;
+}
+
+const worker: ModuleWorker<Env> = {
```

Comment:
Define the `ModuleWorker`, passing thru the `Env` interface as the generic's type argument.

Doing this allows for strict type checks (and auto-complete) for all `env` parameter usage throughout the worker.